### DISCUSSION TITLE
Form Validation

### DISCUSSION THREAD

**Discussion by GoldieBowyien** [GoldieBowyien/Movie_streaming#8] [Q&A] [answered]
How do you create a form to validate users information ?

**Comment by pheneus** [comment, marked as answer]
A data type check confirms that the data entered has the correct data type. For example, a field might only accept numeric data. If this is the case, then any data containing other characters such as letters or special symbols should be rejected by the system. So you can use this as one of the form validation methods

**Reply by GoldieBowyien** [reply]
Thanks Pheneus, will check that out.

**Reply by NeoDev99** [reply]
Thank you bro. It's very informative.
